src/Spectre.libClassifier/ObservationExtractor.h

```diff
@@ -18,22 +18,37 @@ limitations under the License.
 */
 
 #pragma once
+#include <vector>
 #include "Spectre.libDataset/IReadOnlyDataset.h"
 #include "Spectre.libDataset/Empty.h"
 #include "Spectre.libClassifier/OpenCvDataset.h"
-#include "Spectre.libGenetic/Individual.h"
 
 namespace spectre::supervised {
 
 using DataPointer = const spectre::core::dataset::IReadOnlyDataset<Observation, Label, spectre::core::dataset::Empty>*;
 
+/// <summary>
+/// Class used for getting dataset from individual.
```

Comment:
Concept of `Individual` does not exist in the context of `libClassifier`.

Reply:
done